app/Services/CampaignFollowService.php

```diff
@@ -47,6 +47,8 @@ public function remove(Campaign $campaign, User $user): bool
         if (empty($follow)) {
             return false;
         }
+        $campaign->follower--;
```

Comment:
This is a good place to have this code, however if someone creates or deletes a follow through the API (not possible yet, but part of a future ticket), CampaignFollowService isn't called, so this number won't be updated. Which means it's cleaner to add it in the `app\Observers\CampaignFollowerObserver::deleted()` method, so that we aren't linked to calling the service in the future.